extensions/amp-sticky-ad/amp-sticky-ad.md

```diff
@@ -38,11 +38,12 @@ limitations under the License.
 ## Behavior
```

Comment:
Please link to https://ampbyexample.com/components/amp-sticky-ad/ in the beginning of the page.